packages/fiori/src/Bar.js

```diff
@@ -181,6 +181,16 @@ class Bar extends UI5Element {
 	onBeforeRendering() {
 		// Next row is specific for IE11. Please remove after stop support and edit css file
 		[...this.startContent, ...this.middleContent, ...this.endContent].forEach(element => element.classList.add("ui5-bar-content"));
+
+		this.startContent.forEach(x => { ResizeHandler.deregister(x, this._handleResizeBound); }, this);
```

Comment:
If we used se it there, we lose the case in which we add slot after we initially create the bar. This is because the onEnter or onExitDom are related to the bar itself and not to the slots. It will work in the case where we create the bar with all of the slots then make the resize, but not in other cases, which additions are made after the bar has entered the dom.